core/base/test/TString.cxx

```diff
@@ -0,0 +1,12 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+#include "TString.h"
+
+TEST(TString)
+{
+	TString n("testNeg", -1);
+	TString p("testPos", 1);
+	EXPECT_STREQ("", n);
+	EXPECT_STREQ("te", p);
```

Comment:
Wouldn't we expect it to be `t`?

Reply:
Oh, I thought the first index was 0. I'll change it.